crates/madara/client/rpc/src/versions/user/v0_8_0/api.rs

```diff
@@ -54,8 +55,16 @@ pub struct GetStorageProofResult {
 
 #[versioned_rpc("V0_8_0", "starknet")]
 pub trait StarknetWsRpcApi {
-    #[subscription(name = "subscribeNewHeads", unsubscribe = "unsubscribe", item = NewHead, param_kind = map)]
-    async fn subscribe_new_heads(&self, block_id: BlockId) -> jsonrpsee::core::SubscriptionResult;
+    #[subscription(name = "subscribeNewHeads", unsubscribe = "unsubscribeNewHeads", item = NewHead, param_kind = map)]
```

Comment:
```suggestion
    #[subscription(name = "subscribeNewHeads", unsubscribe = "unsubscribe", item = NewHead, param_kind = map)]
```

We want the call to unsubscribe to be `starknet_unsunscribe` as per the specs.

Reply:
I cannot have the same name for unsubscribe for both methods. This seems strange because during an unsubscribe, the ID is provided, so there shouldn't be any conflict.

Reply:
Actually, the problem lies with our `versioned_rpc` macro